integration-test/src/test/java/com/datastrato/gravitino/integration/test/util/spark/SparkTableInfoChecker.java

```diff
@@ -78,6 +83,43 @@ public SparkTableInfoChecker withBucket(
     return this;
   }
 
+  public SparkTableInfoChecker withHourPartition(List<String> partitionColumns) {
+    Transform hourTransform = Expressions.hours(partitionColumns.get(0));
```

Comment:
why just get the first column?

Reply:
partition functions only support passing in one column.

Reply:
ok, i will fix it.

Reply:
done